parts/common/components.json

```diff
@@ -343,15 +358,29 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/azure/secrets-store/provider-azure",
-          "latestVersion": "v1.6.2",
-          "previousLatestVersion": "v1.5.3"
+          "latestVersion": "v1.6.2"
         }
       ],
       "windowsVersions": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/azure/secrets-store/provider-azure",
-          "latestVersion": "v1.5.3",
```

Comment:
Is it expected to change the value of latestVersion to the value of previousLatestVersion?

Reply:
this was auto-generated by bot and I think the value is wrong.

v1.5.3 should be previous one.